acceptance/bundle/syncroot/dotdot-git/script

```diff
@@ -0,0 +1,6 @@
+# This should error, we do not allow syncroot outside of git repo
```

Comment:
Is there a reason why we won't allow it? I can imagine the case when users want to sync build artifacts which are not part of the git repo as well

Reply:
The idea here is that if people use git, they want reproducibility, so sources should be in git.

build artifacts can be placed inside git root and gitignored.